### DISCUSSION TITLE
Access to Admin UI

### DISCUSSION THREAD

**Discussion by svua** [telekom-security/tpotce#945] [General]
Hi there,

How can I access to Admin UI?

I used debian 10 with Post-Install User in Hetzner cloud.
During installation, I set Standard instalation and set a username and password for the web user.

When I use it in Admin UI:
`Wrong user name or password `

Also I tried root access with same password.
I set password for root user and tried with it.
I tried using user tpot and that didn't work either.

I can access to T-Pot Landing Page with web user account. But when I choose kibana or Elasticsearch It just doesn't work. I am getting errors like this

Kibana
```
Fetching listing failed

The dashboard listing could not be fetched: getaddrinfo ENOTFOUND elasticsearch.
```
Elasticsearch
`cluster health: not connected`

**Comment by svua** [comment]
I solved the problem with accessing the Admin UI. The installer doesn't create the tsec user for some reason.
I created it and set a password. After that, I managed to enter the Admin UI.
Also, the system requirements in the documentation are incorrect. After I increase the memory to 16 gigabytes, elastic and kibana started.

**Reply by fkoussen** [reply]
I have the same problem today with Hetzner-Cloud Server Debian11, 16GB RAM, 160GB - thx for the solution with manually adding user tsec - this works now!
